### ISSUE TITLE
10 free message per day

### ISSUE THREAD

**Issue by RamyGomaa** [ading2210/poe-api#194]
Hello, in your README.md you wrote 

"Note that, on free accounts, Claude+ (a2_2) has a limit of 3 messages per day and GPT-4 (beaver) has a limit of 1 message per day. Claude-instant-100k (c2_100k) is completely inaccessible for free accounts. For all the other chatbots, **there seems to be a rate limit of 10 messages per minute**."

it looks like in the past it was 10 messages per minute, but now it's 10 messages per day. it's on every bot now, even the ones that you create.

 is there a way to bypass this?

![image](https://github.com/ading2210/poe-api/assets/56408808/23d9b179-9dc7-4014-bbc0-e9de495b1504)

![image](https://github.com/ading2210/poe-api/assets/56408808/2f2afe1c-4993-4b45-b8b8-0fe7c29848aa)

**Comment by ading2210** [comment]
I haven't experienced a 10 per day rate limit on any of the free bots (gpt-3.5-turbo/claude-instant based). Which bots are these rate limits being applied on?

**Comment by ading2210** [comment]
@RamyGomaa What region are you in? I have a feeling this may be due to some form of region-locking. Where I am in the USA there isn't a daily rate limit on free bots.

![image](https://github.com/ading2210/poe-api/assets/71154407/57fd1d96-fa8a-4944-a9a7-54a701412b36)

**Comment by RamyGomaa** [comment]
> @RamyGomaa What region are you in? I have a feeling this may be due to some form of region-locking. Where I am in the USA there isn't a daily rate limit on free bots.
> 
> ![image](https://user-images.githubusercontent.com/71154407/255653030-57fd1d96-fa8a-4944-a9a7-54a701412b36.png)

I'm From Egypt, I will try to create a new account with a VPN and I will keep you updated. Thanks for the information

**Comment by ading2210** [comment]
@RamyGomaa Does your new account have daily limits on the free models?

**Comment by ading2210** [comment]
@huyremy Does this happen regardless of whether or not the account has been used with poe-api?

**Comment by claell** [comment]
Maybe also related to usage of VPN for setup? @huyremy

**Comment by claell** [comment]
So, you used poe-api with all 5 accounts? The question is whether this also happens on newly created accounts.

**Comment by claell** [comment]
That is not answering the question, I think. Did you use poe-api on all 5 accounts? And were the limits (10 messages per day) there before you used poe-api on the accounts, or only after?

**Comment by ading2210** [comment]
So I assume this is a recent policy change with Poe, not some sort of shadowban due to poe-api detection.

**Comment by claell** [comment]
That's also what I think. Let's see, maybe we get more data points. Seems not too hard to circumvent, when it happens, though.

**Comment by ading2210** [comment]
This issue will probably need further investigation. I don't think its as simple as a region lock or VPN detection.

**Comment by claell** [comment]
Yup, I agree.

**Comment by RamyGomaa** [comment]
> 

all my accounts now have rate limits except the one I created using a VPN in the USA, so I'm inclined to think it's a regional issue. I even created new ones without VPN and they also had rate limits, unfortunately. 

The account I created using a VPN was a temp mail made with "EmailOnDeck" I don't have access to it now but I can send more than 10 messages without a problem with the FORM_KEY and CHAT_TOKEN I got from it. And it's not been banned till now

**Comment by RamyGomaa** [comment]
> This issue will probably need further investigation. I don't think its as simple as a region lock or VPN detection.

all my accounts now have rate limits except the one I created using a VPN in the USA, so I'm inclined to think it's a regional issue. I even created new ones without VPN and they also had rate limits, unfortunately.

The account I created using a VPN was a temp mail made with "EmailOnDeck" I don't have access to it now but I can send more than 10 messages without a problem with the FORM_KEY and CHAT_TOKEN I got from it. And it's not been banned till now
